wgpu-types/src/lib.rs

```diff
@@ -538,6 +538,65 @@ impl Default for Limits {
     }
 }
 
+/// Lists various ways the underlying platform does not conform to the WebGPU standard.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct DownlevelProperties {
+    /// The device supports compiling and using compute shaders.
+    pub compute_shaders: bool,
```

Comment:
should these be flags?

Reply:
Up to you. The latest thing we discussed was because they are passive they should be boolean members, but I don't really have strong feelings.

Reply:
oh, sorry, that's not what my reasoning was last time.
I thought the decision was to whether to include them in `Features` or not, it wasn't about the form of inclusion.
They shouldn't be in features, but they can be in their own `DownlevelFlags` thing, which would live inside `DownlevelProperties`.
Anyway, I don't feel strongly either way.

Reply:
Ahh I get you. I think that's a reasonable balance, I'll fix it in the morn.